### PR TITLE
feat(detail): add localized 'last updated' date

### PR DESCRIPTION
**Pull request by d-loose**
UDENG-1048

Logic to be refined later together with the rest of the snap metadata.

### CODE REVIEW

File: lib/src/detail/detail_page.dart [hunk 1]
```diff
@@ -67,6 +70,12 @@ class _SnapView extends ConsumerWidget {
           value: context.formatByteSize(
               channelInfo?.size ?? model.storeSnap!.downloadSize!)
         ),
+      (
+        label: l10n.detailPageLastUpdatedLabel,
+        value: DateFormat.yMd(
+                Platform.environment['LC_TIME'] ?? Platform.localeName)
```
Comment (jpnurmi) [inline lib/src/detail/detail_page.dart:76]: is there any intl function that could be called on startup to initialize things in one place?

**Review by jpnurmi — APPROVED** [review]
LGTM :+1:

File: lib/src/detail/detail_page.dart (hunk 1 above)
Comment (d-loose) [reply lib/src/detail/detail_page.dart:76]: Hm, I don't quite understand the [docs](https://pub.dev/packages/intl#date-formatting-and-parsing):
 > Note that before doing any DateTime formatting for a particular locale, you must load the appropriate data by calling.
 ```dart
import 'package:intl/date_symbol_data_local.dart';
...
initializeDateFormatting('de_DE', null).then(formatDates);
```

`DateFormat.yMd()` still needs to be called with the correct locale as an argument, though

Comment (jpnurmi) [reply lib/src/detail/detail_page.dart:76]: It probably won't respect `LC_TIME` specifically but there's a helper function like this in `ubuntu_localizations` (originates from UDI):

```dart
/// Initializes [Intl.defaultLocale] for formatting dates, times, and numbers.
///
/// If [locale] is not specified, the system locale is used.
///
/// See also:
/// * [Intl.defaultLocale]
/// * [Intl.systemLocale]
Future<void> initDefaultLocale([String? locale]) async {
  Intl.defaultLocale = locale ?? await findSystemLocale();
}
```

**Pull request merged**